plugins/org.jboss.tools.jst.web.kb/src/org/jboss/tools/jst/web/kb/internal/validation/ELValidator.java

```diff
@@ -510,7 +510,7 @@ private void validateElOperand(ELReference elReference, ELInvocationExpression o
 		if(token==null) {
 			WebKbPlugin.getDefault().logError("The token from unresolved segment is null. EL: [" + operand + "]");
 		}
-		varName = token.getText();
+		varName = token != null ? token.getText() : null;

```

Comment:
Actually. I believe that adding "return null;" in the end of "if(token==null){}" body (after the error message is logged) would be better as we don't need to worry of the rest operations in this case (and actually not able to do anything more).